client/components/List/List.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropsWithChildren } from 'react';
+
+interface Props {
+  variant?:
+  | 'none'
+  | 'with--bullets'
```

Comment:
The classes were written in this way to simulate some kind of BEM syntax - http://getbem.com/introduction/

However, being a prop we can get rid of `with--` and just leave the second part `none`, `bullets`, `checkmark`...